src/Init/Data/Ord.lean

```diff
@@ -37,3 +37,10 @@ instance : Ord Bool where
 
 instance : Ord String where
   compare x y := compareOfLessAndEq x y
+
+def USize.cmp (a b : USize) : Ordering := compare a.val.val b.val.val
+
+instance : Ord USize       := ⟨USize.cmp⟩
```

Comment:
Weird whitespace. Also, I think it's fine to define this inline with `where` as below.

Reply:
done